.github/workflows/gradle-publish.yml

```diff
@@ -120,15 +120,15 @@ jobs:
         uses: gradle/actions/setup-gradle@v4
         
       - id: get-version
-        uses: entur/gha-maven-central/.github/actions/get-version@v1
+        uses: entur/gha-maven-central/.github/actions/get-version@deepenHistoryForTag
         with:
           version_strategy: ${{ inputs.version_strategy }}
           version_file_name: ${{ inputs.version_file_name }}
           version_file_variable_name: ${{ inputs.version_file_variable_name }}
           version_tag_prefix: ${{ inputs.version_tag_prefix }}
 
       - id: increment-version
-        uses: entur/gha-maven-central/.github/actions/increment-version@v1
+        uses: entur/gha-maven-central/.github/actions/increment-version@deepenHistoryForTag
```

Comment:
If you change it back to 'v1', the workflow will use the latest version of the action after the merge.